Dockerfile

```diff
@@ -68,7 +66,6 @@ RUN mkdir -p /etc/symbolicator /data && \
 EXPOSE 3021
 
 COPY --from=symbolicator-build /usr/local/bin/symbolicator /bin
-COPY --from=symbolicator-build /opt/symbolicator-debug.zip /opt/symbolicator.src.zip /opt/
```

Comment:
this might be relevant for self-hosted users. TBH, I have 0 insight into self-hosted works right now, or if it even sets up something equivalent to S4S. If so, where will self-hosted users ever get the symbolicator debug files?

Reply:
Maybe Github release artifacts? I see `craft.yml` references stuff like `symbolicator-Linux-x86_64-debug.zip` which is built by `.github/workflows/build.yml`

Reply:
cc @chadwhitacre

Reply:
@hubertdeng123 Can you take a look here?

Reply:
I'm not too familiar with this either but I think @joshuarli is probably right. Is might be good to double check that self-hosted works with the new workflow